fedimint-server/src/net/p2p_connector.rs

```diff
@@ -40,7 +32,7 @@ pub trait IP2PConnector<M>: Send + Sync + 'static {
 
     async fn connect(&self, peer: PeerId) -> anyhow::Result<DynP2PConnection<M>>;
 
-    async fn listen(&self) -> P2PConnectionListener<M>;
+    async fn accept(&self) -> anyhow::Result<(PeerId, DynP2PConnection<M>)>;
```

Comment:
OK, let's see if I can spot all the changes.

So we merge listening and accepting. :+1: